### ISSUE TITLE
Since update deck tracker is not recognizing new decks

### ISSUE THREAD

**Issue by SerPounc2** [HearthSim/Hearthstone-Deck-Tracker#4018]
Prior to the recent update deck tracker would correctly detect new decks when you entered the play screen and now it doesn't recognize them, even when you try to import decks from constructed.

**Comment by Namisar** [comment]
Experiencing this issue as well. The tracker can still be used to launch hearthstone and the overlay works correctly but it is not importing decks automatically anymore. Entering the 'play' menu now does nothing. Didn't change any settings on my end besides updating to the newest version. The app is correctly telling me that my collection was 'Last updated 1 day ago' referring to when I was running the previous version. Is there anyway to force an update on my collection. I've tried deleting all my old decks and uninstalling/reinstalling, rebooting computer etc. etc. No dice. Read some similar issues from 4 years ago concerning the tracker not being able to create the proper log files. I've confirmed that the logs are already there.
